linux_pipeline/Jenkinsfile_developer_patch_validation

```diff
@@ -9,7 +9,9 @@ pipeline {
     parameters {
         string(defaultValue: "git://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git", description: 'Kernel tree repo', name: 'KERNEL_GIT_URL')
```

Comment:
change commit message to:
[developer] Added LISA validation stage to developer pipeline